src/registry/migrations/0011_includeimplementationlanguage_objects.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 4.2.3 on 2023-08-29 17:59
+
+from django.db import migrations, models
+
+
+def create_initial_objects(apps, schema_editor):
+    ImplementationLanguage = apps.get_model("registry", "ImplementationLanguage")
+
+    languages_and_icons = [
```

Comment:
NOTE: I've added those by hand to ensure these objects will be on database when migrations will be done in production

Reply:
Wouldn't be a good Idea to create a management command to fill the language table when starting from scratch?